src/Components/Filter/Filter.jsx

```diff
@@ -0,0 +1,65 @@
+import './filter.css'
+const Filter=()=>{
+    return(
+        <div className="flex-column">
+                    <button className="btn-link padding-sm">Clear All Filters</button>  
+                    <hr/>            
+                    <h3 className="padding-sm">Filters</h3>
+                    <h3 className="padding-sm">Price Range</h3>
+                    <input className="padding-sm" type="range" min="1" max="100" value="50"/>
+                    <p className="padding-sm">Price under: 500</p>
+                    <h3 className="padding-sm">Category</h3>
+                    <div className="padding-sm">
+                        <input type="checkbox" id="all categories" name="all categories" checked/>
+                        <label for="all categories">All Categories</label>
```

Comment:
we use forName instead of for in react

Reply:
Are you sure? Isn't it `htmlFor`?

Reply:
> Are you sure? Isn't it `htmlFor`?

yes my bad its htmlFor

Reply:
Thankyou for your review comments.  Made the changes